src/pages/LoginPage/LoginPage.js

```diff
@@ -25,6 +25,7 @@ class LoginPage extends Component {
 
   setKey = () => {
     this.props.setApiKey(this.state.apiKey);
```

Comment:
It might be good to also check if api key is valid before saving it and letting user into the app (there's an endpoint to get 'me', i.e. the user that owns api key, if this endpoint returns an error, it means that the api key is not a valid one)